basics/5 terminal/index.md

```diff
@@ -1,27 +1,147 @@
+
+![](desktop.png)
+
+You can now see next to the `%` that I'm at the Desktop. Try running `pwd` to confirm this. Try to find out what folders are located here...
+
+Do notice, however, that any commands you run now are run relative to the Desktop. You cannot, for example, navigate to the Downloads folder by typing `cd Downloads`, as Downloads isn't on the Desktop. You can do so however by typing out the full path: `cd /Users/[yourname]/Downloads` or `cd ~/Downloads` (the `~` always refers to your user directory).
+
+![](downloads.png)
+
+In addition, you can use `..` to refer to one directory higher than your current directory. So when on the Desktop, you could use `cd ..` to return to your user directory. Or even better: use `cd ../Downloads` to navigate directly to Downloads.
+
+> ### Pro tip!
+> Don't feel like typing out a long path to a directory? Simply drag the folder onto your Terminal from your Finder or Explorer. The full path will apear right away!
 
 ## Creating a new repo on Github and cloning
+Now, let's get started using git. Git is a great tool for managing programming projects, from small projects to enterprise-level applications. It keeps track of all changes, and allows for you to work on a project with a team of people, all while avoiding conlicts as much as possible.
+
+Your code in git is placed in a so-called 'repository'. Within a repository, git keeps track of changes and resolved conflicts. Every repository needs to be hosted somehwere. Ofcourse, you could host it on your own device. If you're going to work with other people, however, this won't work. Most repository's, therefore, are primarily hosted through services like [Github](https://github.com). You should already have an account there. If not, create one now!
+
+Once you log in to Github, you end up in your dashboard. From here, you can create a new repository by hitting the 'New' button on the left of the screen.
+
+![](github_dash.png)
+
+Give your repository a name (it must be unique within your account), a description and select whether you want it to be public or private. Public repository's, including all the code in them, can be seen by anyone on the internet. Private repository's can only be seen by the people you add to them. Finally, tick the box next to 'Add a README file'.
+
+![](github_new.png)
+
+You now have a new Github repository. It's empty, apart from the README file. So let's make sure you can actually work on your code within the repository, on your own device. To do so, you must 'Clone' the repository by clicking on the green 'Code' button and copying the URL that appears. 
+
+![github_clone.png]()
+
+Then, in a Terminal window, navigate to a folder where you want your code to live. Then, run `git clone [URL]`, replacing `[URL]` with the URL you just copied. Git should perform some magic, and a new folder with the name of your repository should appear. In there, you'll find the README file from Github!
+
+![](git_clone.png)
```

Comment:
Let op dat we ze niet aanleren git-repos in de "Downloads" dir te dumpen. 
Een `~/dev` aanmaken speciaal voor git/development.